### PR TITLE
tests: cmsis_rtos_v2: mark as non-SMP-safe test

### PR DESCRIPTION
**Pull request by IRISZZW**
tests/portability/cmsis_rtos_v2 is non-SMP-safe, we need to turning the other CPU off when running this test.

similar to #13704 #13705 #13706

### CODE REVIEW

**Review by abrodkin — APPROVED** [review]

**Comment by evgeniy-paltsev** [comment]
Hi @IRISZZW ,
could you pleas add some details about what exactly is done in not SMP safe way in this test?

Thanks.

File: tests/portability/cmsis_rtos_v2/prj.conf [hunk 1]
```diff
@@ -16,3 +16,5 @@ CONFIG_CMSIS_V2_THREAD_DYNAMIC_MAX_COUNT=10
 CONFIG_TIMEOUT_64BIT=n
 CONFIG_CMSIS_V2_THREAD_MAX_STACK_SIZE=1024
 CONFIG_CMSIS_V2_THREAD_DYNAMIC_STACK_SIZE=1024
+
+CONFIG_SMP=n
```
Comment (nashif) [inline tests/portability/cmsis_rtos_v2/prj.conf:20]: instead, set the number of cpus to 1, this should have the same effect.

**Review by nashif — CHANGES_REQUESTED** [review]
I think this can be closed, it has already been addressed.

**Comment by nashif** [comment]
obsoleted by another PR.

**Pull request closed without merging**